src/shared/parse.h

```diff
@@ -54,4 +54,47 @@ int Parse_FreeSourceHandle(int handle);
 bool Parse_ReadTokenHandle(int handle, pc_token_t *pc_token);
 int Parse_SourceFileAndLine(int handle, char (&filename)[MAX_QPATH], int *line);
 
+/*
+===============
+Parse_CSV
```

Comment:
I did a similar comment in IRC, with this link: https://sebsauvage.net/wiki/doku.php?id=csv sorry it's french, but basically it explains how “full” CSV support is very hard, that's why given we just do a very small subset of CSV, we may just call it otherwise and don't pretend to do CSV.